app/views/pages/routes/show.html.erb

```diff
@@ -26,7 +26,7 @@
 
         <p class="govuk-body"><%= t(".any_other_answer.will_continue_to", next_question_number: route_summary_card_data_presenter.next_page&.position) %></p>
 
-        <% if FeatureService.new(group: current_form.group).enabled? :branch_routing %>
+        <% if FeatureService.new(group: current_form.group).enabled?(:branch_routing) && !route_summary_card_data_presenter.routes.first&.exit_page_heading.present? %>
```

Comment:
Is it worth having a method, like `is_exit_page?` instead of checking for exit_page_heading like this?

Reply:
Sorted, used the one you've added